joomla/joomla-cms
# convert icon-trash to fas fa-trash

Pull Request for Issue # .

### Summary of Changes
changes the trash icon in actions dropdown to "fas fa-trash"


### Testing Instructions
apply pr.
inspect action dropdown "trash" icon.
verify icon class is "fas fa-trash"

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/91083332-be4f7e00-e60f-11ea-8ab4-72e6e694b190.png)

### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/91083268-9c55fb80-e60f-11ea-8067-4757622a9307.png)

### Documentation Changes Required
none
